### PR TITLE
Bug 1233103: Added possibility to remove translations by the locale m…

### PR DESCRIPTION
**Pull request by jotes**
Hi @Osmose, @mathjazz could you look at this? I thought that locale managers should have ability to remove translations. However, we can close this ticket if you think otherwise.

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ var Pontoon = (function (my) {
                       (this.approved ? this.approved_user ?
                         'Approved by ' + this.approved_user : '' : 'Approve') +
                       '"></button>' +
-                      ((self.user.email && (self.user.email === this.email)) ? '<button class="delete fa" title="Delete"></button>' : '') +
+                      ((self.user.email && (self.user.email === this.email) || self.user.isLocaleManager) ? '<button class="delete fa" title="Delete"></button>' : '') +
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: We should actually allow all translators to remove translations, so replace `isLocaleManager` with `isTranslator`.

Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: And that means this is all we need to change in this PR. You can revert other changes.

Comment (jotes) [inline pontoon/base/static/js/translate.js]: Done, could you look again?

On Thu, Dec 17, 2015 at 2:48 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/base/static/js/translate.js
> <https://github.com/mozilla/pontoon/pull/308#discussion_r47906607>:
>
> > @@ -107,7 +107,7 @@ var Pontoon = (function (my) {
> >                        (this.approved ? this.approved_user ?
> >                          'Approved by ' + this.approved_user : '' : 'Approve') +
> >                        '"></button>' +
> > -                      ((self.user.email && (self.user.email === this.email)) ? '<button class="delete fa" title="Delete"></button>' : '') +
> > +                      ((self.user.email && (self.user.email === this.email) || self.user.isLocaleManager) ? '<button class="delete fa" title="Delete"></button>' : '') +
>
> And that means this is all we need to change in this PR. You can revert
> other changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/308/files#r47906607>.
>



-- 
Kind regards

​Jarek "jotes" Śmiejczak,
​P​
ython programmer​ by day, mozillian by heart..
Homepage: http://joyfulhack.in, Github: http://github.com/jotes
Mobile: +48693027040, Mozillian: http://mozillians.org/u/jotes

**Comment by mathjazz** [comment]
Much simpler now. ;)

r+

**Pull request merged**